Kernel/Arch/aarch64/MMU.cpp

```diff
@@ -1,197 +0,0 @@
-/*
```

Comment:
I think this code is really useful to integrate into the `MemoryManager`, and shouldn't just be dropped like this. I think it is pretty harsh to just drop the work of James on the floor.

Reply:
It’s still available in the git history.